CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 
 - Support for Wazuh 4.7.2.
 
+### Changed
+
+- Updated the use case: Give a user permissions to read and manage a group of agents with minimum configuration reading permissions. ([#6761](https://github.com/wazuh/wazuh-documentation/pull/6761))
```

Comment:
What use case? And the PR reference doesn't look correct

Reply:
Good catch